main.go

```diff
@@ -56,6 +56,7 @@ func generate(request pluginpb.CodeGeneratorRequest) (*pluginpb.CodeGeneratorRes
 	disableKubeMarkers := false
 
 	var messagesWithEmptySchema []string
+	var ignoredKubeMarkers []string
```

Comment:
ignoredKubeMarkersRegex ?

Reply:
Kinda sorta, a list of substrings to identify kube markers which need to be ignored